src/hotspot/cpu/ppc/vm_version_ppc.cpp

```diff
@@ -142,13 +111,8 @@ void VM_Version::initialize() {
     if (FLAG_IS_DEFAULT(UseCharacterCompareIntrinsics)) {
       FLAG_SET_ERGO(UseCharacterCompareIntrinsics, true);
     }
-    if (SuperwordUseVSX) {
-      if (FLAG_IS_DEFAULT(UseVectorByteReverseInstructionsPPC64)) {
-        FLAG_SET_ERGO(UseVectorByteReverseInstructionsPPC64, true);
-      }
-    } else if (UseVectorByteReverseInstructionsPPC64) {
-      warning("UseVectorByteReverseInstructionsPPC64 specified, but needs SuperwordUseVSX.");
-      FLAG_SET_DEFAULT(UseVectorByteReverseInstructionsPPC64, false);
+    if (FLAG_IS_DEFAULT(UseVectorByteReverseInstructionsPPC64)) {
+      FLAG_SET_ERGO(UseVectorByteReverseInstructionsPPC64, true);
```

Comment:
I think this part should be reverted. It is still possible to switch off SuperwordUseVSX.